Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.21
+
+RUN go install github.com/cloudentity/cac@dev
```

Comment:
use multi-state docker build https://medium.com/geekculture/optimizing-golang-docker-images-with-multi-stage-builds-ca7b305faa for better caching and small target docker size and use `go build` instead of `go install` so you can build docker from a branch as well. 

example

```
FROM golang:1.21 AS build

WORKDIR /app

COPY go.mod go.sum ./
RUN go mod download
COPY . .

RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .

FROM alpine:latest

WORKDIR /app

COPY --from=build /app/app .

CMD ["./app", "start"]
```